src/Form/SitepackageType.php

```diff
@@ -44,34 +44,32 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label' => 'Base Package',
                 'choices' => [
                     'Bootstrap Package' => 'bootstrap_package',
-                    'Fluid Styled Content' => 'fluid_styled_content',
+                    'Minimal Site Package' => 'fluid_styled_content',
                 ],
                 'expanded' => true,
             ])
             ->add('typo3Version', ChoiceType::class, [
                 'label' => 'TYPO3 Version',
                 'choices' => [
-                    '13.4 (Beta)' => 13.4,
-                    '12.4' => 12.4,
-                    '11.5' => 11.5,
-                    '10.4' => 10.4,
-                    '9.5' => 9.5,
-                    '8.7' => 8.7,
+                    '13.4 (stable)' => 13.4,
```

Comment:
The BETA flag was meant to be for the generated packages and was not about the current TYPO3 version - i would prefer not to change it and only remove the BETA from v13

Reply:
Ack, will change that